### DISCUSSION TITLE
How to get grafana dashboards from each prometheus operator release

### DISCUSSION THREAD

**Discussion by vishiy** [prometheus-operator/kube-prometheus#1689] [General]
I am looking for how to get the dashboards that gets packaged with every version of prometheus operator (which is used by kube-prometheus-stack chart). I am PRWing prometheus metrics to a remote store (cloud) and using managed grafana from a cloud provider, and i want to import those dashboards into managed grafana for the version of operator i am using. I know i can export & import manually from the grafana instance installed by the k-p-stack, but i am trying to make it more sustainable by not going that route, but rather get the .json files packaged by operator for each of the operator release, so that i can import those dashboards to my managed grafana instance. Any help is appreciated. Thanks in advance.

**Comment by bryanasdev000** [comment]
I think there are a few way to do that:

- Compile jsonnet with different release targets
- Get from the default generated manifests navigating by tags (example for v0.9.0 https://github.com/prometheus-operator/kube-prometheus/blob/v0.9.0/manifests/grafana-dashboardDefinitions.yaml)
- Get it from Helm project (but without your possible modifications) (example for v34.3.0 https://github.com/prometheus-community/helm-charts/tree/kube-prometheus-stack-34.3.0/charts/kube-prometheus-stack/templates/grafana/dashboards-1.14)
